ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/product/v1/product/clientlib/js/variantSelector.js

```diff
@@ -45,6 +45,10 @@ class VariantSelector {
             button.addEventListener('click', this._onSelectVariant.bind(this));
         }, this);
 
+        this._state.useUid = this._element
```

Comment:
I get an error here, which is caused by `this._element.querySelector(VariantSelector.selectors.quantity)` returning `null`

```
clientlib-cif.lc-8a1ac571e177293e65b2067774853a93-lc.min.js:1030 Uncaught TypeError: Cannot read properties of null (reading 'hasAttribute')
    at new VariantSelector (clientlib-cif.lc-8a1ac571e177293e65b2067774853a93-lc.min.js:1030:89)
    at HTMLDocument.onDocumentReady (clientlib-cif.lc-8a1ac571e177293e65b2067774853a93-lc.min.js:1209:29)
```